tests/rules/match-rules.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+function f($filename) {
+    // ruleid: test
+    bad("convert $filename");
+}
```

Comment:
Maybe this is not needed, but we could have a test to show that tainting works in the presence of the new constructs.